src/server/services/wmo/wmo.test.ts

```diff
@@ -102,4 +102,76 @@ describe('Transform api items', () => {
       `);
     });
   });
+
+  describe('getDisclaimer', () => {
+    const baseAanvraag: ZorgnedAanvraagTransformed = {
+      titel: 'Test Voorziening',
+      isActueel: true,
+      datumEindeGeldigheid: null,
+      datumIngangGeldigheid: null,
+      datumAanvraag: '2024-01-01',
+      datumBesluit: '2024-01-01',
+      datumBeginLevering: '2024-01-01',
+      datumEindeLevering: '2024-01-01',
+      datumToewijzing: '2024-01-01',
+      datumOpdrachtLevering: '2024-01-01',
+      leverancier: 'Test Leverancier',
+      leveringsVorm: 'Test LeveringsVorm',
+      productsoortCode: 'Test ProductSoortCode',
+      productIdentificatie: 'Test ProductIdentificatie',
+      resultaat: 'toegewezen',
+      betrokkenen: [],
+      documenten: [],
+      id: 'test-id',
+    };
+
+    const baseAanvragen = [baseAanvraag];
+
+    it('should return undefined when no matching conditions', () => {
+      const result = getDisclaimer(baseAanvraag, baseAanvragen);
+      expect(result).toBeUndefined();
+    });
+
+    it('should return disclaimer for actuele voorziening with matching eerdere voorziening', () => {
+      const aanvraag = {
+        ...baseAanvraag,
+        isActueel: true,
+      };
+
+      const aanvragen = [
+        aanvraag,
+        {
+          ...baseAanvraag,
+          isActueel: false,
+          datumEindeGeldigheid: '1-11-2024',
+        },
+      ];
+
+      const result = getDisclaimer(aanvraag, aanvragen);
+      expect(result).toBe(
+        'Dit hulpmiddel staat per ongeluk ook bij "Eerdere en afgewezen voorzieningen". Daar vindt u het originele besluit met de juiste datums.'
+      );
+    });
+
+    it('should return disclaimer for eerdere voorziening with matching actuele voorziening', () => {
+      const aanvraag = {
+        ...baseAanvraag,
+        isActueel: false,
+      };
+
+      const aanvragen = [
+        aanvraag,
+        {
+          ...baseAanvraag,
+          isActueel: true,
+          datumIngangGeldigheid: '31-10-2024',
+        },
+      ];
+
+      const result = getDisclaimer(aanvraag, aanvragen);
+      expect(result).toBe(
+        'Door een fout staat dit hulpmiddel ten onrechte bij Eerdere en afgewezen voorzieningen. Kijk bij "Huidige voorzieningen" of in de brief bovenaan.'
```

Comment:
moeten hier ook quotes bij of juist niet....heb het letterlijk overgenomen maar er zijn twee smaakjes.
Ik zou zeggen in beide gevallen quotes maar benieuwd naar jullie mening